### DISCUSSION TITLE
Increase Quality and Low framerate

### DISCUSSION THREAD

**Discussion by Mlebailly22** [Motion-Project/motion#1657] [Q&A]
Hi,
I am beginning in motion, and i try to make a camera for my 3D Printer.
But with my first try the quality is a disaster, i know i have a bad camera (a Pi Camera V2), but on the stream i think that clearly worse than the normal. I think i have some settings that i need to change in my conf files, but i don't know which one. 
Can you help me to find which settings i need to change, to have the best picture available, even if i must turn down my framerate to 1 picture / seconde, i don't care.

Thank you very much

**Comment by Mr-Dave** [comment]
The V2 camera is not bad.  You'll need to provide more details on what you've changed from the default and how the quality is being judged.

**Comment by Mlebailly22** [comment]
Great i'm happy for the camera then, so it's just settings to change. 

For the moment i only change few settings :

width : 960

height: 720

rotate : 180

framerate : 5 (i try to lower the framerate to increase the quality but it's maybe a bad idea)

And there is the result : 
![image](https://user-images.githubusercontent.com/50553103/227803279-25be6fac-94b1-43c7-af22-24b2d996c3ab.png)

as you can see, my picture look zoomed, and a very low resolution.

Thanks for your help
